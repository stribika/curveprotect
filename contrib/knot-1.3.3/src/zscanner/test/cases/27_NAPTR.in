$ORIGIN	.
$TTL	1

; OK
@	NAPTR	0	0	""	""	""	@	; The simplest variant
@	NAPTR	65535	65535	""	""	""	@	; Maximal numbers
@	NAPTR	65535	65535	""	""	"!^urn:cid:.+@([^\.]+\.)(.*)$!\\2!i"	@	; Regexp example
@	NAPTR	0	0	""	""	""	\0320\ \\\"\.\@\*.tld.	; Dname with specials
@	TYPE35	\# 8 00000000000000 00				; TYPE + Hexadecimal rdata
@	TYPE35	0	0	""	""	""	@	; TYPE
@	naptr	0	0	""	""	""	@	; Type in lower-case

; KO
@	NAPTR
@	NAPTR							; Empty rdata
@	NAPTR	\# 0						; Hex empty rdata
@	NAPTR	\#						; Missing hex length
@	NAPTR	65536	0	""	""	""	@	; Order overflow
@	NAPTR	0	65536	""	""	""	@	; Preference overflow
@	NAPTR	0	0	""	""	""	@  x	; Unexpected item
@	NAPTR	0	0	""	""	""		; Missing item
@	NAPTR	\# 9 00000000000000 00 00			; Too long rdata
@	NAPTR	\# 9 00000000000000 00				; Bad rdata length
