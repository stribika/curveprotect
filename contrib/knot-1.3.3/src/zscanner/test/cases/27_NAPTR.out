OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0023
RDATA=00000000000000 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0023
RDATA=FFFFFFFF000000 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0023
RDATA=FFFFFFFF00001F215E75726E3A6369643A2E2B40285B5E2E5D2B2E29282E2A2924215C322169 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0023
RDATA=00000000000000 082030205C222E402A03746C6400
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0023
RDATA=00000000000000 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0023
RDATA=00000000000000 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0023
RDATA=00000000000000 00
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_ENUMBER16_OVERFLOW
------
WARNG=ZSCANNER_ENUMBER16_OVERFLOW
------
WARNG=ZSCANNER_EBAD_REST
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_HEX_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
